.github/workflows/runtime-tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Runtime Tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tests:
+    name: Runtime tests
+    runs-on:
+    - ubuntu-latest
+    concurrency:
+      group: runtime-tests-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Prepare a test environment
+      run: |
+        cd integration-tests/runtime-tests
+        docker-compose -f docker-compose-ci.yml up -d
+
+    - name: Wait when the node will be started
+      timeout-minutes: 5
+      run: |
+        cd integration-tests/runtime-tests
+        time ../../scripts/wait_for_polkadot_launch.sh
+
+    - name: Run tests
+      run: |
+        cd integration-tests/runtime-tests
+        docker run --rm -v$(pwd):/build -e "ENDPOINT=composable:9988" --workdir /build --network runtime-tests_default --entrypoint=/build/DockerRun.sh node:17
+
+    - name: Cleanup
+      run: |
+        cd integration-tests/runtime-tests
+        docker-compose -f docker-compose-ci.yml down --remove-orphans
```

Comment:
Can we add this to check.yaml, as we want these to pass for each PR

Reply:
can we make it next time? or optional? 
i think these run long and likely will give false positives.

Reply:
> Can we add this to check.yaml, as we want these to pass for each PR

this check depends on composable-sandbox docker image
we should build a new composable-sandbox docker image for every PR if we want to do this. so this will increase check time by about 1h15m

ps: I will add a depend between the tests and composable-sandnox in CI in one of next PRs.

Reply:
> can we make it next time? or optional? i think these run long and likely will give false positives.

as a creation of files with weights? using a tag?